src/controller/appController.ts

```diff
@@ -0,0 +1,46 @@
+import { Request, Response } from 'express';
+
+import * as userServices from '../services/userServices';
+import * as listServices from '../services/listServices';
+import * as cardServices from '../services/cardServices';
+import * as boardServices from '../services/boardServices';
+
+export const board = async (req: Request, res: Response) => {
+  const result = await boardServices.createBoard(req.params.id, req.body);
+  res.status(result.status).json(result);
+};
+
+export const createCardController = async (req: Request, res: Response) => {
```

Comment:
If you separate a file you can simply name it as `create`. Separating file works as namespacing such that you can do this:

```js
import * as cardController from './controller/cards';

cardController.create()
cardController.fetchaAll()
```

It the same convention for services, routes, model if that's possible